packages/decdk/package.json

```diff
@@ -134,7 +134,8 @@
     "jsii-reflect": "^0.10.5",
     "jsonschema": "^1.2.4",
     "yaml": "1.5.0",
-    "yargs": "^13.2.2"
+    "yargs": "^13.2.2",
+    "@aws-cdk/aws-route53-targets": "^0.31.0"
```

Comment:
Hardcore nitpicking/bikeshedding, but I'd really like it if we could keep this list sorted alphabetically. Feel free to ignore though (it's not so important in the grand scheme of things).